Database and DAO

DDOwzw opened the pull request:
Hi team,

This is our first version DAO and MySQL connection code.

We still lack of user insert, update, and delete functionality right now. It will be published in later version.

Now all the available functions are:
Site: find, addOrUpdate, delete;
Manager: find, addOrUpdate, delete;
WashMachine: find, add, update, addOrUpdate, delete, list all;
User: findById, findByUsername;

lisz1012 commented on src/main/java/com/cinderella/dao/DBConnection.java:
```diff
@@ -182,6 +184,16 @@
 	public void deleteManagerById(int EmployeeAccountNumber, boolean onCascade) throws Exception;
+	
+	/**
+	 * Get all the washing machines of the address.
```
Please add a comment about the only address in DB

DDOwzw commented:
Changed user class datatype for balance and phone number. Modified tableCreate and MySQLConnection respectively. Added comment for default address info in the list machine API interface.
